iib/workers/tasks/opm_operations.py

```diff
@@ -907,3 +907,224 @@ def verify_operator_exists(
             is_package_in_index = True
             log.info("operator package %s found in index_db %s", operator_package, index_db_path)
     return is_package_in_index, index_db_path
+
+
+def set_opm_verison(index_image: str) -> str:
+    """
+    This function will verify if opm config is set in worker
+    if the config entry exists for the index image version
+    and return the opm version path to be used in the entire IIB request.
+    :param str index_image: from_index_image for the request
+    :return str opm_path: opm_path to be used for the entire request
```

Comment:
I do not see any returns. This function just sets the global variable but does not return anything (which is correct).